README.md

```diff
@@ -345,6 +345,16 @@ the attribute type does not have any special semantic or effect currently.
 As already mentioned, all this configuration parameters can also be adjusted using the
 [`config.js`](https://github.com/telefonicaid/IoT-STH/blob/develop/config.js) file whose contents are self-explanatory.
 
+It is important to note that there is a limitation on the bytes a MongoDB namespace (concatenation of the database name and
```

Comment:
By introducing this paragraph it should be asumed that the data model is fixed to `collection-per-entity`, right? I mean, we still maintain the possibility to switch to another data model, and AFAIK this hashing mechanism is only addressing the `collection-per-entity` data model.

I've been thinking about fixing the data model and so on, and I think we can hide from the documentation any reference to the `data_model` parameter, defaulting it to `collection-per-entity` but maintaining the code that allows to change the data model for internal testing purposes (I'm affraid we decide to fix the data model and in the future the bosses decide for another data model :)).